src/main/java/app/BookOld.java

```diff
@@ -9,7 +9,7 @@
 
 @Data
 @AllArgsConstructor
-public class Book {
+public class BookOld {
     String autor;
```

Comment:
This "BookOld" is an unfortunate name... Please consider changing it in the future!